sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,119 @@
+# validate sudoku
```

Comment:
Switch to `"""` style commenting for top of file comments.

Reply:
Done